docs/build/collator/how-to-stake-in-zeitgeist.md

```diff
@@ -75,7 +85,7 @@ candidates" step)
 
 5.The amount that you would like to stake (in Pennock, which means you need to
 multiply 10^10, so if you want to stake 1 ZTG, you should fill in is
-`10000000000`)
+`10000000000`).
```

Comment:
This dot is inconsistent to every other bullet point that does not end with that symbol. I suggest to leave it away at all or to use it in any case.